source/developer/webhooks-incoming.rst

```diff
@@ -89,16 +90,16 @@ Tips and Best Practices
 
 5. The external application may be written in any programming language as long as it supports sending an HTTP POST request in the required JSON format to a specified Mattermost URL.
 
-6. For the HTTP request body, if `Content-Type` is specified as ``application/json`` in the header of the HTTP request, then the body can be direct JSON. For example,
+6. For the HTTP request body, if ``Content-Type`` is specified as ``application/json`` in the header of the HTTP request, then the body can be direct JSON. For example,
 
 .. code-block:: text
 
   {"text": "Hello, this is some text."}
 
-7. When using the ``icon_emoji`` parameter - the user profile image is replaced by the emoji provided. This will also override the ``icon_url`` parameter if both are provided. 
+7. When using the ``icon_emoji`` parameter - the user profile image is replaced by the emoji provided. This will also override the ``icon_url`` parameter if both are provided.
```

Comment:
```suggestion
7. When using the ``icon_emoji`` parameter, the user profile image is replaced by the emoji provided. This will also override the ``icon_url`` parameter if both are provided.
```